### ISSUE TITLE
Expected [title] to be a GraphQL nullable type

### ISSUE THREAD

**Issue by dangodev** [yarax/swagger-to-graphql#90]
In typeMap.js:116:

```js
const type = jsonSchema.required && jsonSchema.required.includes(propertyName) ? graphql.GraphQLNonNull(baseType) : baseType;
```

`graphql.GraphQLNonNull(baseType)` errs if you try and feed it an `object`. It tries to run `GraphQLNonNull(SomeLongObject_title)` and throws an error.

---

Added a [PR](https://github.com/yarax/swagger-to-graphql/pull/91) to fix the issue, along with a test to verify.

**Comment by stvvt** [comment]
I've been playing with official docker [swagger file](https://docs.docker.com/engine/api/v1.37/swagger.yaml). When tried to just convert it to GraphQL type definitions
```
$ swagger-to-graphql --swagger=./swagger.yaml
Error: Expected Image_RootFS! to be a GraphQL nullable type.
    at invariant (/home/stv/projects/research/docker-graphql-api/node_modules/graphql/jsutils/invariant.js:19:11)
    at assertNullableType (/home/stv/projects/research/docker-graphql-api/node_modules/graphql/type/definition.js:322:50)
    at new GraphQLNonNull (/home/stv/projects/research/docker-graphql-api/node_modules/graphql/type/definition.js:288:19)
    at Object.GraphQLNonNull (/home/stv/projects/research/docker-graphql-api/node_modules/graphql/type/definition.js:290:12)
    at /home/stv/projects/research/docker-graphql-api/node_modules/swagger-to-graphql/lib/typeMap.js:139:94
...
```
I've also submit a [PR](https://github.com/yarax/swagger-to-graphql/pull/105)

**Comment by 0xR** [comment]
Fixed by #105
